containerz/containerz.proto

```diff
@@ -359,31 +342,6 @@ message Volume {
   bool read_only = 3;
 }
 
-// A device to be added to the container.
```

Comment:
Is your client up-to-date? It looks like this was what was added in #264 being removed.

Reply:
oops good catch